code/game/objects/items/shields.dm

```diff
@@ -67,58 +37,79 @@
 		if(0 to 25)
 			. += span_warning("It's falling apart!")
 
-/obj/item/shield/riot/proc/shatter(mob/living/carbon/human/owner)
-	playsound(owner, 'sound/effects/glassbr3.ogg', 100)
-	new /obj/item/shard((get_turf(src)))
+/obj/item/shield/proc/shatter(mob/living/carbon/human/owner)
```

Comment:
just make this the dropped material, breaking sound, and material you use to fix variables, dont do this overriding stuff